src/setupTests.js

```diff
@@ -0,0 +1,4 @@
+import Enzyme from 'enzyme';
```

Comment:
this file also isn't needed - it is for React apps. The starter code was a little confusing that way.